ZukoAzula2016/src/org/team1540/zukoazula/IntakeArm.java

```diff
@@ -0,0 +1,26 @@
+package org.team1540.zukoazula;
+
+import ccre.channel.BooleanIO;
+import ccre.channel.BooleanInput;
+import ccre.channel.EventInput;
+import ccre.channel.FloatInput;
+import ccre.channel.FloatOutput;
+import ccre.ctrl.ExtendedMotor;
+import ccre.ctrl.ExtendedMotorFailureException;
+import ccre.drivers.ctre.talon.TalonExtendedMotor;
+import ccre.frc.FRC;
+
+public class IntakeArm {
+    private static final TalonExtendedMotor intakeArmCAN = FRC.talonCAN(9);
+    private static final TalonExtendedMotor intakeArmRollerCAN = FRC.talonCAN(11);
+    private static final FloatInput encoder = intakeArmCAN.modEncoder().getEncoderVelocity();
```

Comment:
Ah, sorry, I left it in for later, when we'll want to limit the range of motion based on the encoder. (And yes, it shouldn't be velocity.)